### PR TITLE
test: keep LF line endings so tests pass on Windows

### PR DESCRIPTION
**Pull request by stevetaggart**
Git on Windows can automatically convert LF to CRLF when you clone a repo. 
This ensures that the LF line endings are kept.

#### ✔ Checklist:

- [x] Commit messages have been written in [Conventional Commits](https://www.conventionalcommits.org) format;

### CODE REVIEW

**Review by DDtKey — DISMISSED** [review]

**Review by DDtKey — COMMENTED** [review]
Might be better to add this to the global `.gitattribures`?

**Comment by mgrachev** [comment]
> Might be better to add this to the global `.gitattribures`?

Agree 👍

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1,4 @@
 *.rs diff=rust
 *.md diff=markdown
+*.env text eol=lf
+*.env.golden text eol=lf
```
Comment (DDtKey) [inline .gitattributes:4]: Add newline at end of the file, please 🙂 
```suggestion
*.env.golden text eol=lf

```

Comment (stevetaggart) [reply .gitattributes]: Good catch. I'm not sure why my editor didn't add that based on the `.editorconfig` file.

**Review by DDtKey — APPROVED** [review]

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@stevetaggart Thank you for your help 👍

🙏 If it’s not difficult for you, please support the project - star on GitHub ⭐️

**Pull request merged**